pkg/pillar/types/locationconsts.go

```diff
@@ -4,21 +4,62 @@
+	AADirname = TmpDirname + "/AssignableAdapters"
+	// DNCDirname - directory for per-model Network Config files
+	DNCDirname = TmpDirname + "/DeviceNetworkConfig"
+
+	// PersistDir - Location to store persistent files.
 	PersistDir = "/persist"
-	// PersistRktDataDir - persist rkt dir used by rkt
-	PersistRktDataDir = PersistDir + "/rkt"
-	// RwImgDirname - location for images
-	RwImgDirname = PersistDir + "/img" // We store images here
-	// DownloadDirname - Location for downloader to download images
+	// RwImgDirname - We store images here
+	RwImgDirname = PersistDir + "/img"
+	// DownloadDirname - Location of downloaded images
 	DownloadDirname = PersistDir + "/downloads"
-	// ImgCatalogDirname - Location for App Images
-	ImgCatalogDirname = DownloadDirname + "/" + appImgObj
-	// VerifiedDirname - Read-only images named based on sha256 hash each
-	// in its own directory
+	// ObjectDownloadDirname - Location of downloaded objects
+	ObjectDownloadDirname = PersistDir + "/downloads"
+	// CertificateDirname - Location of certificates
+	CertificateDirname = PersistDir + "/certs"
+	// CheckpointDirname - location of config checkpoint
+	CheckpointDirname = PersistDir + "/checkpoint"
+	// ImgCatalogDirname - location of downloaded app images. Read-only images
+	// named based on sha256 hash each in its own directory
+	ImgCatalogDirname = DownloadDirname + "/" + AppImgObj
+	// PendingDirname - location of app images in pending state
+	PendingDirname = ImgCatalogDirname + "/pending"
+	// FinalDirname - location of verified app images
+	FinalDirname = ImgCatalogDirname + "/verified"
+	// VerifiedDirname - Location of verified App images. Read-only images
+	// named based on sha256 hash each in its own directory
 	VerifiedDirname = ImgCatalogDirname + "/verified"
+
+	// PersistRktDataDir - Location of rkt dir,
+	PersistRktDataDir = PersistDir + "/rkt"
+	// PersistRktLocalConfigDir - Location of dir used by rkt local data
+	PersistRktLocalConfigDir = PersistDir + "/rktlocal"
+	// PersistRktLocalConfigAuthDir - used for storing auth files by rkt
+	PersistRktLocalConfigAuthDir = PersistRktLocalConfigDir + "/auth.d"
+
+	// IdentityDirname - Config dir
+	IdentityDirname = "/config"
+	// SelfRegFile - name of self-register-filed file
+	SelfRegFile = IdentityDirname + "/self-register-failed"
+	// ServerFileName - server file
+	ServerFileName = IdentityDirname + "/server"
+	// DeviceCertName - device certificate
+	DeviceCertName = IdentityDirname + "/device.cert.pem"
+	// OnboardCertName - Onboard certificate
+	OnboardCertName = IdentityDirname + "/onboard.cert.pem"
+	// OnboardKeyName - onboard key
+	OnboardKeyName = IdentityDirname + "/onboard.key.pem"
+
+	// AppImgObj - name of app image obj dir
+	AppImgObj = "appImg.obj"
+	// BaseOsObj - name of base image obj dir
+	BaseOsObj = "baseOs.obj"
+	// CertObj - Name of Certificate obj. dir
+	CertObj = "cert.obj"
+	// ConfigDir - name of Config dir.
+	ConfigDir = "/config"
```

Comment:
But you call this IdentityDirname as well? Pick one?

Reply:
I followed the existing code.. I thought it was intentional - to have 2 variables - identityDir and ConfigDir..  if not, I would pick "ConfigDir" as the name ( and not IdentityDir )..